ui/src/app/shared/header/header.component.css

```diff
@@ -26,3 +20,10 @@
   justify-content: space-between;
   align-items: center;
 }
+
+.chipShell {
+  width: fit-content !important;
```

Comment:
I'm guessing the `!important` is needed, but would be great if it weren't. Otherwise maybe add a comment here to explain why it is needed.

Reply:
Actually only the `padding = 0` is necessary, removed